src/server/zen-remote-server.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
+exec_prefix="${prefix}"
+libdir="${prefix}/lib"
+includedir="${prefix}/include"
```

Comment:
These are broken (do not respect the actual configured installation directories).

It's a pity that CMake doesn't support something like https://mesonbuild.com/Pkgconfig-module.html#pkggenerate

Reply:
Thanks for your comment!
I am very happy as this is the first contribution from someone other than my peers.

Regarding your comment, it needs to be corrected, I have set up an issue.
https://github.com/zigen-project/zen-remote/issues/8

By the way, may I ask why you are interested in this pull request or project?

Reply:
I am a buildsystems nerd with an interest in interoperability and platform-defined correctness. Things like the manual work needed for https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html and the lack of proper pkg-config generators are pet peeves of mine, and pkg-config files in particular seem to be gotten wrong by the majority of people that don't auto-generate them.

It *is* fixable, see e.g. https://github.com/jtojnar/cmake-snips/

I do stumble across a variety of projects, at any rate.